[imperihome] Uses persistence APIs that need to be adapted for openHAB 3

The add-ons port build (https://ci.openhab.org/job/openHAB-Addons-port/) fails on imperihome.io with

```
[ERROR] /home/jenkins/jenkins-agent1/workspace/openHAB-Addons-port/bundles/org.openhab.io.imperihome/src/main/java/org/openhab/io/imperihome/internal/handler/DeviceHistoryHandler.java:[110,4552] The constructor HistoryItem(ZonedDateTime, Number) is undefined
```

The bundle will therefore be removed from that build and once the code is moved to the master branch, it needs to be adapted to the new API.

--- a/bundles/org.openhab.io.imperihome/src/main/java/org/openhab/io/imperihome/internal/handler/DeviceHistoryHandler.java
+++ b/bundles/org.openhab.io.imperihome/src/main/java/org/openhab/io/imperihome/internal/handler/DeviceHistoryHandler.java
@@ -110,7 +110,7 @@ public class DeviceHistoryHandler {
                 State state = historicItem.getState();
                 if (state instanceof DecimalType) {
                     Number value = ((DecimalType) state).toBigDecimal();
-                    resultItems.add(new HistoryItem(historicItem.getTimestamp(), value));
+                    resultItems.add(new HistoryItem(historicItem.getTimestamp().toInstant().toEpochMilli(), value));
                 }
             }
 

--- a/bundles/pom.xml
+++ b/bundles/pom.xml
@@ -19,6 +19,7 @@
   <modules>
     <!-- io -->
     <module>org.openhab.io.homekit</module>
+    <module>org.openhab.io.imperihome</module>
     <module>org.openhab.io.neeo</module>
     <module>org.openhab.io.openhabcloud</module>
     <module>org.openhab.io.transport.modbus</module>

